kubernetes_deploy/production/deployment.yml

```diff
@@ -40,6 +40,26 @@ spec:
             secretKeyRef:
               name: sentry
               key: dsn
+        - name: SECRET_KEY
+          valueFrom:
+            secretKeyRef:
+              name: secret-key
+              key: SECRET_KEY
```

Comment:
Is it possible to make the production application fail to boot without the required secrets?

Reply:
Done